bin/logstash.bat

```diff
@@ -47,5 +50,29 @@ set JAVA_OPTS=%LS_JAVA_OPTS%
 
 rem jruby launcher will pickup JAVA_OPTS set above to set the JVM options before launching jruby
 %JRUBY_BIN% "%LS_HOME%\lib\bootstrap\environment.rb" "logstash\runner.rb" %*
+goto end
+
+:version
+set "LOGSTASH_VERSION_FILE1=%LOGSTASH_HOME%/logstash-core/versions-gem-copy.yml"
```

Comment:
should this be `LS_HOME` instead of `LOGSTASH_HOME`? This is the first reference to `LOGSTASH_HOME` I see in this file, but `LS_HOME` is referenced several times.

Reply:
Thanks. Fixed.